src/entities/stabilityTracker/ui/StabilityTrackerItem.tsx

```diff
@@ -125,11 +124,11 @@ const StabilityTrackerItemScreen = (props: Props) => {
 
   const handleGyroscopeError = useCallback(
     (error: Error) => {
-      toast.show(error.toString());
+      Toast.show({ type: 'toast', text1: error.toString() });
```

Comment:
I think we could make the use of useBanner hook here

```tsx
banner.show(error.toString(), { type: 'toast' });
```

Reply:
I've update to use this to banners but would like to keep it differentiate between banner (presented at top) and toast (presented at bottom in dark)